build.gradle

```diff
@@ -11,13 +11,11 @@ plugins {
   id 'net.researchgate.release' version '2.7.0'
 }
 
-// defaultTasks 'build', 'checkLicenses', 'javadoc'
 defaultTasks 'build'
 
 def buildAliases = ['dev': [
     'spotlessApply',
     'build',
-//    'checkLicenses',
```

Comment:
Can you add this line back in? It was commented out so gradle will build in [this commit](https://github.com/PegaSysEng/artemis/commit/3b88f251c1f9b71cc75247f73dba36ed25dd0e6c). This check will need to be fixed. :)

Reply:
fixed it :-)